system/ee/ExpressionEngine/Controller/Design/System.php

```diff
@@ -132,9 +132,28 @@ public function edit($template_id)
             'author' => (empty($author)) ? '-' : $author->getMemberName(),
         );
 
+        $vars['buttons'] = [
+            [
+                'name' => 'submit',
+                'type' => 'submit',
+                'value' => 'save',
+                'text' => 'save',
+                'working' => 'btn_saving'
+            ],
+            [
+                'name' => 'submit',
+                'type' => 'submit',
+                'value' => 'save_and_close',
+                'text' => 'save_and_close',
+                'working' => 'btn_saving'
+            ]
+        ];
+
         $this->loadCodeMirrorAssets();
 
-        ee()->view->cp_page_title = sprintf(lang('edit_template'), lang($template->template_name));
+        ee()->view->cp_page_title = sprintf(lang($template->template_name));
```

Comment:
`sprintf` is missing parameter here. I think we could just remove it and keep only `lang($template->template_name)`

Reply:
changed